client/styles/app.css

```diff
@@ -1,4 +1,6 @@
 .container {
+  color: rgb(72, 72, 72);
```

Comment:
Set color using HEX value instead of to keep consist with other stylesheets, or edit other style sheets to use RBG value

Reply:
Good catch, I'll make those changes!